src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -36,6 +51,127 @@ public class ParserUtilTest {
 
     private static final String WHITESPACE = " \t\r\n";
 
+    @Test
+    public void parseDateTime_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> ParserUtil.parseDateTime((String) null));
+    }
+    @Test
+    public void parseDateTime_invalidInput_throwsParseException() {
```

Comment:
is the tests for parsing for date redundant because the the Time class already does these checks when it is being constructed

Reply:
I thought these unit tests were implemented to confirm that we have already done these checks in the time class?

Reply:
Oops I comment on the wrong line. I meant in the ParserUtil class itself